### DISCUSSION TITLE
deploying php with ext-ffi and ext-igbinary

### DISCUSSION THREAD

**Discussion by tacman** [dokku/dokku#6487] [Q&A]
First, thanks to everyone who makes dokku happen, what a great tool.

I have an php 8 application that needs to ffi and igbinary extensions installed.  With heroku, there's [documentation](https://elements.heroku.com/buildpacks/heroku/heroku-buildpack-php) for how to do this:

heroku config:set HEROKU_PHP_PLATFORM_REPOSITORIES="https://customrepo.com/packages.json?composer-repository-only=ext-igbinary,ext-redis"

Is there a way to do this with dokku?

Thanks.

**Comment by josegonzalez** [comment]
You can also set those environment variables via `dokku config:set`.

**Comment by tacman** [comment]
How?  Can php extensions be set with env variables?

**Reply by josegonzalez** [reply]
The command:

```shell
heroku config:set
```

Is the same as:

```shell
dokku config:set
```

The only difference is that you need to specify your app name:

```shell
# replace $APP_NAME with your apps name
dokku config:set $APP_NAME HEROKU_PHP_PLATFORM_REPOSITORIES="https://customrepo.com/packages.json?composer-repository-only=ext-igbinary,ext-redis"
```
